spec/scripts/benchmarks.sh

```diff
@@ -5,15 +5,17 @@
 #
 # Usage: ./spec/scripts/benchmarks.sh jruby:9.1
 #
-set -exuo pipefail
+set -exo pipefail
 
-if [ $# -lt 1 ]; then
-  echo "Arguments missing"
-  exit 2
-fi
-
-RUBY_IMAGE=${1}
+RUBY_IMAGE=${1:?"missing RUBY IMAGE"}
 VERSION=$(echo "${RUBY_IMAGE}" | cut -d":" -f2)
+REFERENCE_REPO=${2}
+
+if [ -z "${REFERENCE_REPO}" ] ; then
```

Comment:
The reason for this particular condition is that he git reference repo is required to be mounted in the docker container when using the CI pipeline. But it should also allow running the scripts locally without that particular reference repo.